HeishaMon/htmlcode.h

```diff
@@ -539,6 +539,16 @@ static const char webCSS[] PROGMEM =
   ".w3-btn { margin-bottom:10px; }"
   ".heishatable { display: none; }"
   "#cli{ background: black; color: white; width: 100%; height: 400px!important; }"
+  "/* Style additions by @fbloemhof */"
+  "#cli, textarea { height: 75vh!important; width: 100%; }"
+  "table { margin-bottom: 10px; }"
+  "@media (prefers-color-scheme: dark) { html, div.w3-container.w3-center.heishatable, #cli, textarea, input, button, select { background-color: #23262a; color: #e1e1e1 }"
```

Comment:
Why don't you enclose them all in one single `@media` rule?

Reply:
I don't completely understand how the htmlcode.h file is build. Is it just rendering everything in the "" to the browser?

Reply:
Yes

Reply:
So this would give the same result? 

```
  "@media (prefers-color-scheme: dark) {"
  "html, div.w3-container.w3-center.heishatable, #cli, textarea, input, button, select { background-color: #23262a; color: #e1e1e1 }"
  "div.w3-container.w3-left { color: #e1e1e1 }"
  ".w3-theme, .w3-sidebar {color:#e1e1e1 !important; background-color:#000000 !important}"
  ".w3-red, .w3-hover-red:hover { color: #e1e1e1!important; background-color: #616161!important; }"
  ".w3-table-all tr:nth-child(odd){background-color:#23262a}.w3-table-all tr:nth-child(even){background-color:#2f3338}"
  ".w3-bordered tr,.w3-table-all tr{border-bottom:1px solid #000000}.w3-striped tbody tr:nth-child(even){background-color:#000000}"
  ".w3-table,.w3-table-all{border-collapse:collapse;border-spacing:0;width:100%;display:table}.w3-table-all{border:1px solid #000} }"
```

Based on the rest of the content of the `htmlcode.h` file, I think my first attempt is more clear. But I'm happy to change it if you like.

Reply:
If you would move the PR to the igorybema repo, a image is build for each PR so you can test things yourself.

Reply:
The workflow is also working at my own repo, I currently have a build running with this PR.

Reply:
Makes it easier to test.

Reply:
please agree with @CurlyMoo for final solution and I will approve then, now closing PR